Dockerfile

```diff
@@ -1,25 +1,40 @@
-FROM node:14 AS builder
-WORKDIR /hydra-build
+FROM node:16-alpine AS node
+
+FROM node AS node-with-gyp
+RUN apk add g++ make python3
+
+FROM node-with-gyp AS builder
+WORKDIR /squid
 ADD package.json .
 ADD package-lock.json .
 RUN npm ci
 ADD tsconfig.json .
 ADD src src
 RUN npm run build
 
-
-FROM node:14 AS processor
-WORKDIR /hydra-project
+FROM node-with-gyp AS deps
+WORKDIR /squid
 ADD package.json .
 ADD package-lock.json .
-RUN npm ci # TODO: --production
-COPY --from=builder /hydra-build/lib lib
+RUN npm ci --production
+
+FROM node AS squid
+WORKDIR /squid
+COPY --from=deps /squid/package.json .
+COPY --from=deps /squid/package-lock.json .
+COPY --from=deps /squid/node_modules node_modules
+COPY --from=builder /squid/lib lib
 ADD db db
-ADD manifest.yml .
 ADD schema.graphql .
-ADD .env .
+# TODO: use shorter PROMETHEUS_PORT
+ENV PROCESSOR_PROMETHEUS_PORT 3000
+EXPOSE 3000
+EXPOSE 4000
```

Comment:
You use `4350` in `.env`. Isn't that an issue?

Reply:
I think it's better not to change it. Also copied from [template](https://github.com/subsquid/squid-template). Deployment works well with it